packages/@o3r/eslint-config-otter/package.json

```diff
@@ -51,22 +51,26 @@
   },
   "peerDependencies": {
     "@angular-devkit/schematics": "~18.2.0",
-    "@angular-eslint/builder": "~18.3.0",
+    "@angular-eslint/builder": "~18.4.0",
+    "@angular-eslint/eslint-plugin": "~18.4.0",
+    "@angular-eslint/eslint-plugin-template": "~18.4.0",
+    "@angular-eslint/template-parser": "~18.4.0",
     "@angular/compiler": "~18.2.0",
     "@o3r/eslint-plugin": "workspace:^",
     "@o3r/schematics": "workspace:^",
     "@schematics/angular": "~18.2.0",
     "@stylistic/eslint-plugin-ts": "^2.7.0",
-    "angular-eslint": "~18.3.0",
-    "eslint": "^8.57.0 || ^9.0.0",
+    "@typescript-eslint/eslint-plugin": "^7.14.1",
+    "@typescript-eslint/parser": "^7.14.1",
+    "@typescript-eslint/utils": "^7.14.1",
+    "eslint": "^8.57.0",
```

Comment:
This one did not introduce a breaking change, why do you remove the support for 9^ ?